mycroft/client/speech/listener.py

```diff
@@ -240,23 +249,25 @@ class RecognizerLoop(EventEmitter):
                 (optional, can be set later via self.bind )
     """
 
-    def __init__(self, bus, watchdog=None, stt=None):
+    def __init__(self, bus, watchdog=None, stt=None, fallback_stt=None):
         super(RecognizerLoop, self).__init__()
         self._watchdog = watchdog
         self.mute_calls = 0
         self.stt = stt
+        self.fallback_stt = fallback_stt
```

Comment:
Consider handling this as a list of fallbacks for multi-language support

Reply:
multilingual should just pass the lang argument to execute method? i dont think we want to load local models per language, and if we do maybe a dedicated plugin for this is better? see vosk for a plugin where i recently added multi lang support

keep in mind that in ovos lang can come from wakeword and is passed along to STT unlike mycroft lang which is static

Reply:
Right, but each lang may require a different plugin, right? For local use that might be too heavy though, so maybe the solution is some remote STT endpoint for something like a server where you are more likely to have multiple languages being transcribed concurrently